### ISSUE TITLE
u3d/install & u3d/available no INI file error on Linux

### ISSUE THREAD

**Issue by IvanRibakov** [DragonBox/u3d#242]
### Issue Checklist

- [x] I'm using the latest u3d version
- [x] I ran `u3d --help`
- [x] I read the [README](https://github.com/DragonBox/u3d/blob/master/README.md)
- [x] I made sure that a similar [issue](https://github.com/DragonBox/u3d/issues) doesn't exit

### Issue Description

Running `u3d install 2017.1.1f1` results in 

```
root@a1bb72ffaccd:/# u3d install 2017.1.1f1 --trace
No INI file for version 2017.1.1f1. Try discovering the available versions with 'u3d available -f'
/usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/unity_version_definition.rb:44:in `available_packages': undefined method `keys' for nil:NilClass (NoMethodError)
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/unity_version_definition.rb:48:in `available_package?'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/commands.rb:232:in `block in verify_package_names'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/commands.rb:231:in `each'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/commands.rb:231:in `verify_package_names'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/commands.rb:122:in `install'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/commands_generator.rb:156:in `block (2 levels) in run'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/commander-4.4.4/lib/commander/command.rb:178:in `call'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/commander-4.4.4/lib/commander/command.rb:153:in `run'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/commander-4.4.4/lib/commander/runner.rb:446:in `run_active_command'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/commander-4.4.4/lib/commander/runner.rb:68:in `run!'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/commander-4.4.4/lib/commander/delegates.rb:15:in `run!'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/commands_generator.rb:209:in `run'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/lib/u3d/commands_generator.rb:37:in `start'
	from /usr/local/rvm/gems/ruby-2.4.1/gems/u3d-1.0.15/exe/u3d:7:in `<top (required)>'
	from /usr/local/rvm/gems/ruby-2.4.1/bin/u3d:23:in `load'
	from /usr/local/rvm/gems/ruby-2.4.1/bin/u3d:23:in `<main>'
	from /usr/local/rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
	from /usr/local/rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
```

Tried above command on versions `2017.2.0b11`, `2017.1.1f1` and `5.6.3f1` without any luck.

Similar happens when trying to list available versions AND their packages:

```
root@a1bb72ffaccd:/# u3d available -f -p
Fetching central 'versions.json' cache
Version 5.1.0f3: http://download.unity3d.com/download_unity/unity-editor-installer-5.1.0f3+2015091501.sh
No INI file for version 5.1.0f3. Try discovering the available versions with 'u3d available -f'
Packages:
error: undefined method `each_key' for nil:NilClass. Use --trace to view backtrace
```

Am I missing something obvious again or has the location of ini files changed on the network?

**Comment by lacostej** [comment]
@IvanRibakov Unity doesn't provide INI files for Linux, we create them. Maybe (most probably) I broke something when introducing the cache. I don't use yet the Linux version daily so I may have forgotten to clear our cache when running our tests.

**Comment by IvanRibakov** [comment]
Trick with `/etc/hosts` worked, thanks!

**Comment by lacostej** [comment]
@IvanRibakov feature released as part of 1.0.16
